### PR TITLE
Added issue templates

### PR DESCRIPTION
**Pull request by toddysm**
Signed-off-by: Toddy Mladenov <toddysm@gmail.com>

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug-or-issue.yaml [hunk 1]
```diff
@@ -0,0 +1,70 @@
+# Copyright The Notary Project Authors.
```
Comment (yogeshbdeshpande) [inline .github/ISSUE_TEMPLATE/bug-or-issue.yaml:1]: Please change the first line to match other Community files within Veraison

Comment (toddysm) [reply .github/ISSUE_TEMPLATE/bug-or-issue.yaml:1]: I am trying to find an example for the text to use. Will the following work:

> Copyright The Mozilla Maintainers and Contributors and The Veraison Go-COSE Authors

or just 

> Copyright The Veraison Go-COSE Authors

Comment (SteveLasker) [reply .github/ISSUE_TEMPLATE/bug-or-issue.yaml:1]: Thanks @toddysm 

> Copyright The Veraison Go-COSE Authors

As this project as now decoupled from the Mozilla maintainers.

Comment (toddysm) [reply .github/ISSUE_TEMPLATE/bug-or-issue.yaml]: Fixed the copyright notice.

File: .github/ISSUE_TEMPLATE/config.yml [hunk 2]
```diff
@@ -0,0 +1,16 @@
+# Copyright The Veraison Go-COSE Authors.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
```
Comment (SteveLasker) [inline .github/ISSUE_TEMPLATE/config.yml:6]: Same MPL issue: https://github.com/veraison/go-cose/blob/main/LICENSE

File: .github/ISSUE_TEMPLATE/bug-or-issue.yaml [hunk 3]
```diff
@@ -0,0 +1,72 @@
+# Copyright The Veraison Go-COSE Authors.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
```
Comment (SteveLasker) [inline .github/ISSUE_TEMPLATE/bug-or-issue.yaml:6]: The project is under MPL: https://github.com/veraison/go-cose/blob/main/LICENSE, based on the original Mozilla owners.

File: .github/ISSUE_TEMPLATE/feature-request.yaml [hunk 4]
```diff
@@ -0,0 +1,65 @@
+# Copyright The Veraison Go-COSE Authors.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
```
Comment (SteveLasker) [inline .github/ISSUE_TEMPLATE/feature-request.yaml:6]: MPL: https://github.com/veraison/go-cose/blob/main/LICENSE

**Review by SteveLasker — CHANGES_REQUESTED** [review]
go-cose is under MPL, based on its history.

File: .github/ISSUE_TEMPLATE/config.yml (hunk 2 above)
Comment (toddysm) [reply .github/ISSUE_TEMPLATE/config.yml]: Updated the license text.

File: .github/ISSUE_TEMPLATE/bug-or-issue.yaml (hunk 3 above)
Comment (toddysm) [reply .github/ISSUE_TEMPLATE/bug-or-issue.yaml]: Updated the license text.

File: .github/ISSUE_TEMPLATE/feature-request.yaml (hunk 4 above)
Comment (toddysm) [reply .github/ISSUE_TEMPLATE/feature-request.yaml]: Updated the license text.

**Review by SteveLasker — APPROVED** [review]
LGTM, thanks @toddysm

File: .github/ISSUE_TEMPLATE/feature-request.yaml [hunk 5]
```diff
@@ -0,0 +1,72 @@
...
+    id: context
+    validations:
+      required: false
+    attributes:
+      label: "Any additional context?"
+      description: "Add any other context or screenshots about the feature request here."
+  - type: markdown
+    attributes:
+      value: |
+        If you want to contribute to this project, we will be happy to guide you through out contribution process especially when you already have a good proposal or understanding of how to imrpove the functionality. Join us at https://slack.cncf.io/ and choose #notary-v2 channel.
```
Comment (yogeshbdeshpande) [inline .github/ISSUE_TEMPLATE/feature-request.yaml:72]: We should not have notary-v2 references here!

File: .github/ISSUE_TEMPLATE/bug-or-issue.yaml [hunk 6]
```diff
@@ -0,0 +1,79 @@
...
+    id: version
+    validations:
+      required: true
+    attributes:
+      label: What is the version of your Notation CLI or Notation Library?
+      description: "For the CLI, you can use the command `notation version` to get it. For the libraries check the `go.mod` file."
+  - type: markdown
+    attributes:
+      value: |
+        If you want to contribute to this project, we will be happy to guide you through out contribution process especially when you already have a good proposal or understanding of how to fix this issue. Join us at https://slack.cncf.io/ and choose #notary-v2 channel.
```
Comment (yogeshbdeshpande) [inline .github/ISSUE_TEMPLATE/bug-or-issue.yaml:79]: We use Verasion Zulipchat for Veraison discussions

**Review by yogeshbdeshpande — COMMENTED** [review]
Kindly address these comments

File: .github/ISSUE_TEMPLATE/bug-or-issue.yaml [hunk 7]
```diff
@@ -0,0 +1,77 @@
...
+      required: true
+    attributes:
+      label: Describe your environment 
+      description: "Notation installation method (e.g. wget, curl, brew, apt-get, yum, chocolate, MSI) if applicable / OS version / Shell type (e.g. zsh, bash, cmd.exe, Bash on Windows) / Golang version if applicable"
+  - type: textarea
+    id: version
+    validations:
+      required: true
+    attributes:
+      label: What is the version of your Notation CLI or Notation Library?
```
Comment (yogeshbdeshpande) [inline .github/ISSUE_TEMPLATE/bug-or-issue.yaml:72]: This is still using Notation CLI. We need to remove this as well!

File: .github/ISSUE_TEMPLATE/bug-or-issue.yaml [hunk 8]
```diff
@@ -0,0 +1,77 @@
...
+      description: "For the CLI, you can use the command `notation version` to get it. For the libraries check the `go.mod` file."
```
Comment (yogeshbdeshpande) [inline .github/ISSUE_TEMPLATE/bug-or-issue.yaml:73]: Same here!

File: .github/ISSUE_TEMPLATE/feature-request.yaml [hunk 9]
```diff
@@ -0,0 +1,70 @@
...
+    id: context
+    validations:
+      required: false
+    attributes:
+      label: "Any additional context?"
+      description: "Add any other context or screenshots about the feature request here."
+  - type: markdown
+    attributes:
+      value: |
+        If you want to contribute to this project, we will be happy to guide you through out contribution process especially when you already have a good proposal or understanding of how to imrpove the functionality. Join us at https://veraison.zulipchat.com/.
```
Comment (yogeshbdeshpande) [inline .github/ISSUE_TEMPLATE/feature-request.yaml:70]:
```suggestion
        If you want to contribute to this project, we will be happy to guide you through out contribution process especially when you already have a good proposal or understanding of how to improve the functionality. Join us at https://veraison.zulipchat.com/.
```

**Review by yogeshbdeshpande — APPROVED** [review]
LGTM!

**Pull request merged**